src/index.js

```diff
@@ -1,10 +1,28 @@
 import express from "express";
-import comments from "./comments";
-import products from "./products";
-import vehicle from "./vehicles";
-import contacts from "./contacts";
+
+import bodyParser from "body-parser";
+import contacstRoutes from "./routes/ContactRoutes";
```

Comment:
You imported these modules, but they do not exist. There should be a file for each of these